lib/Sema/TypeCheckSwitchStmt.cpp

```diff
@@ -604,8 +612,9 @@ namespace {
             // a disjunction of its sub-spaces because nothing got reduced.
             // This is especially helpful when dealing with `unknown` case
             // in parameter positions.
-            if (s1 == reducedSpace)
+            if (s1 == reducedSpace){
```

Comment:
I appreciate the bracing, but please standardize on the spacing in the rest of this file.  If in doubt, run `clang-format`.